src/lucky/renderable.cr

```diff
@@ -209,6 +209,26 @@ module Lucky::Renderable
     file(path, content_type, disposition, filename, status.value)
   end
 
+  def data(
+    data : String,
+    content_type : String = "application/octet-stream",
+    disposition : String = "attachment",
+    filename : String? = nil,
+    status : Int32? = nil
+  ) : Lucky::DataResponse
+    data(IO::Memory.new(data), content_type, disposition, filename, status)
+  end
+
+  def data(
+    io : IO,
+    content_type : String = "application/octet-stream",
+    disposition : String = "attachment",
+    filename : String? = nil,
+    status : Int32? = nil
+  ) : Lucky::DataResponse
+    Lucky::DataResponse.new(context, io.gets_to_end, content_type, disposition, filename, status)
+  end
+
```

Comment:
Maybe we should also add an overload that takes `HTTP::Status` like the others? Just to keep those consistent.